king_phisher/client/dialogs/campaign_selection.py

```diff
@@ -154,7 +207,7 @@ def load_campaigns(self):
 				(hlfg_color if is_expired else fg_color),
 				(html.escape(campaign.description, quote=True) if campaign.description else None)
 			))
-		self.gobjects['label_campaign_info'].set_text("Showing {0:,} Campaign{1}".format(len(store), ('' if len(store) == 1 else 's')))
+		self.gobjects['label_campaign_info'].set_text("{0:,} campaign{1} found in the database".format(len(store), ('' if len(store) == 1 else 's')))
```

Comment:
Is there a way to count the number of campaigns that are being shown? If so this label should be kept the same because I think it would be useful for users to see how their filters are affecting the numbers of campaigns shown. For example if I filter out other users campaigns and it drops by 20 I can tell there are 20 campaigns from other users etc.